### DISCUSSION TITLE
AtGames Legends Mini

### DISCUSSION THREAD

**Discussion by Devedander** [ndeadly/MissionControl#452] [Q&A]
Any chance for support of this?  When I turn it on it never even shows up in controller grip screen even though it's in pairing mode.  

Thanks!

**Comment by ndeadly** [comment]
If it's not showing up (assuming you've done everything correctly) then this can mean one of several things.
- the controller is a Bluetooth LE device (currently not supported)
- the controller uses an incompatible Bluetooth chipset (we see this with a bunch of bootleg controllers)
- the controller is compatible, but pairing is failing for some reason.

We can't do much about those first two currently, but the third one is a possibility.
You can confirm/rule out Bluetooth LE if you connect to a PC and check the device properties. Someone reported another atgames controller as no longer pairing after one of the recent updates, so that's a definite possibility too. I will make a debug build for you later today to log the pairing process and see what's up.

**Reply by Devedander** [reply]
Ok I’ll check when I get home. Thanks!

**Comment by Devedander** [comment]
Dang it's listed as Bluetooth LE Device 212010093a26

I see BTLE is next on the list of things to work on... I guess I just hold my breath? :D

**Reply by ndeadly** [reply]
Bummer. Yeah, BLE should be supported eventually, but until then you'll just have to wait.

**Reply by Devedander** [reply]
Thanks for all your hard work in the mean time!
